src/content/blog/2024-06-20-simple-open-payments-guide.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "A Simple Guide to the Open Payments Standard"
+description: Learn how the Open Payments standard makes online payments easier and more accessible for everyone.
+date: 2024-06-20
+slug: simple-open-payments-guide
+author: Sarah Jones
+author_url: https://www.linkedin.com/in/sarah-jones-ba6bb6b9
+tags:
+  - Interledger
+  - Open Payments
+---
+
+## The Current Digital Payments Landscape
+
+Handling payments is a crucial part of many online applications. Whether you’re running an eCommerce site selling products, a fundraising platform accepting donations, a streaming service charging for content, or a subscription service requiring monthly fees, digital payments are at the core of the operation.
+
+Many application developers rely on third-party payment gateways to handle these transactions. Companies like PayPal, Stripe, and Square offer services to process credit card payments on behalf of the application. In return for these services, applications incur various fees, which can include monthly fees, flat rates per transaction, or a percentage of the transaction amounts. This approach, while convenient, introduces additional expenses and makes the application reliant on a third-party provider. This reliance can limit control over the user experience, such as forcing users to trust third-party providers with sensitive information like credit card details.
```

Comment:
![third-party](https://github.com/interledger/interledger.org-developers/assets/1461498/c7322d29-8cf4-417a-9659-ebbee3d414a0)

Maybe we could label the third-party provider blob with all the various fees as well.

Reply:
to be more explicit, maybe we say "credit card _network_" in the diagram